From e7e7b8ab0b1454e97e5af0a52fdd110dd04efa86 Mon Sep 17 00:00:00 2001
From: Charlie Paul <cpaul.windriver@gmail.com>
Date: Wed, 2 Sep 2015 23:12:58 -0700
Subject: [PATCH 57/86] fs/vmfs: Updated code to support linux 4.1 (continue)

Changed the code to compile with 4.1 and updated design to
support 4.1 design.

Signed-off-by: Charlie Paul <cpaul.windriver@gmail.com>
[Original patch is from linux-yocto-4.1 axxia/base branch.]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 fs/vmfs/Makefile |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/fs/vmfs/Makefile b/fs/vmfs/Makefile
index c96cdd1..fa1d63b 100644
--- a/fs/vmfs/Makefile
+++ b/fs/vmfs/Makefile
@@ -4,11 +4,9 @@
 
 obj-$(CONFIG_VMFS_FS) += vmfsfs.o
 
-#vmfsfs-objs := proc.o dir.o cache.o inode.o file.o ioctl.o getopt.o \
+vmfsfs-objs := proc.o dir.o cache.o inode.o file.o ioctl.o getopt.o \
 		symlink.o messagebox.o msg.o vfs.o
 
-vmfsfs-objs := proc.o dir.o inode.o ioctl.o getopt.o \
-		symlink.o msg.o vfs.o
 
 # If you want debugging output, you may add these flags to the EXTRA_CFLAGS
 # VMFSFS_PARANOIA should normally be enabled.
-- 
1.7.5.4

